pkg/apis/autoscaling/v1alpha1/pa_lifecycle.go

```diff
@@ -140,6 +146,12 @@ func (pas *PodAutoscalerStatus) IsInactive() bool {
 	return cond != nil && cond.Status == corev1.ConditionFalse
 }
 
+// IsActive returns true if the pod autoscaler is Active.
+func (pas *PodAutoscalerStatus) IsActive() bool {
```

Comment:
New methods deserve UTs.

Reply:
no tests yet.  I'd like to see the high-level idea works first.

Reply:
Tests added.